### ISSUE TITLE
What is it for?

### ISSUE THREAD

**Issue by tdekoekkoek** [MSOpenTech/AngularCloudDataConnector#6]
Project sounds interesting, but it is really a bit vague on what the benefits are.  The site could use an explanation of the problems it is trying to solve and how it addresses those problems.

thanks!

**Comment by deltakosh** [comment]
hey Trevor, did you check the landing page? http://acdc.azurewebsites.net/

**Comment by tdekoekkoek** [comment]
Yes I did, thanks.  I just feel there should be some more description of what the benefits are and how it is used.  For example the Facebook provider: is this for building Facebook apps?  Does this offer something that I cannot already achieve with the Facebook SDK?   Still it looks interesting and I hope to try it out.

**Comment by deltakosh** [comment]
I get it! I'm gonna update the site to be clearer :)
